crates/typstyle-core/src/pretty/mod.rs

```diff
@@ -368,7 +371,7 @@ impl<'a> PrettyPrinter<'a> {
                 self.arena.nil()
             },
         )
-        .nest(2)
+        .nest(self.config.tab_spaces as isize)
```

Comment:
should we just define it as isize?

Reply:
The isize is required by pretty, where dedent is allow. 
But in the config, a signed number of spaces is meaningless, IMO.

Reply:
make sense